packages/js-dash-sdk/src/providers/PriorityContextProvider.ts

```
@@ -0,0 +1,331 @@
+    for (const provider of this.providers) {
+      try {
+        if ('isAvailable' in provider.provider) {
+          const available = await (provider.provider as ProviderWithCapabilities).isAvailable();
+          if (available) return provider;
+        } else {
+          // Try a simple operation to check availability
+          await provider.provider.getLatestPlatformBlockHeight();
+          return provider;
+        }
+      } catch {
+        continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)